<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Группа</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Управление группой</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Monotype Corsiva</family>
            <pointsize>18</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Ваши очки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="lcdNumber">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Ваше сообщение:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="font">
           <font>
            <family>Monotype Corsiva</family>
            <pointsize>12</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Отправить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="font">
           <font>
            <family>Monotype Corsiva</family>
            <pointsize>12</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string> От именя сообщества </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Выйти из группы</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Просмотр участников</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
